src/main/java/com/example/codebase/domain/location/dto/LocationCreateDTO.java

```diff
@@ -30,16 +29,13 @@ public class LocationCreateDTO {
 
     private String englishName;
 
-    @NotBlank(message = "링크는 필수입니다.")
-    @Size(max = 255, message = "링크는 최대 255자까지 가능합니다.")
-    private String link;
-
     @NotEmpty(message = "연락처는 필수입니다.")
     private String phoneNumber;
 
-    @NotEmpty(message = "웹사이트 주소는 필수입니다.")
-    private String webSiteUrl;
+  @NotEmpty(message = "웹사이트 주소는 필수입니다.")
+  @Size(max = 255, message = "웹사이트 주소길이는 최대 255자까지 가능합니다.")
```

Comment:
\@NotEmpty와 \@Size 유효성 검증을 한곳에서 통합할 수 있습니다. 
```suggestion
  @Size(min = 1, max = 255, message = "웹사이트 주소길이는 1~255자까지 가능합니다.")
```
는 어떨까요?